packages/fxa-content-server/tests/functional/recovery_key.js

```diff
@@ -126,19 +126,13 @@ registerSuite('Recovery key', {
             )
           )
 
-          // enter old key
+          // enter old key and check for error tooltip
           .then(fillOutRecoveryKey(recoveryKey))
           .then(
             testElementExists(
               selectors.COMPLETE_RESET_PASSWORD_RECOVERY_KEY.TOOLTIP
             )
           )
-          .then(
```

Comment:
It appears that the tooltip takes longer than the maximum page load time to appear? Either way, it doesn't really provide any value to check the text in here if it is failing so much.

Ref https://234255-36831081-gh.circle-artifacts.com/0/screenshots/17d5b9fb.png